diff --git a/AdsLib/CMakeLists.txt b/AdsLib/CMakeLists.txt
index 47f97a4..4908af2 100644
--- a/AdsLib/CMakeLists.txt
+++ b/AdsLib/CMakeLists.txt
@@ -28,7 +28,7 @@ endif()
 
 
 if(WIN32 EQUAL 1)
-    target_link_libraries(ads PUBLIC ws2_32)
+    target_link_libraries(ads PUBLIC ws2_32 wsock32)
 endif()
 
 target_link_libraries(ads PUBLIC Threads::Threads)
diff --git a/CMakeLists.txt b/CMakeLists.txt
index 89846a4..6c4c5c3 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -24,3 +24,11 @@ option(BUILD_SHARED_LIBS "Build using shared libraries" ON)
 add_subdirectory(AdsLib)
 add_subdirectory(AdsLibTest)
 add_subdirectory(example)
+
+install(
+    TARGETS ${PROJECT_NAME}
+    LIBRARY DESTINATION lib
+    ARCHIVE DESTINATION lib
+    RUNTIME DESTINATION bin
+    INCLUDES DESTINATION include
+)
