diff --git a/crypto/CMakeLists.txt b/crypto/CMakeLists.txt
index b1ded85..50f934a 100644
--- a/crypto/CMakeLists.txt
+++ b/crypto/CMakeLists.txt
@@ -1005,7 +1005,7 @@ target_include_directories(crypto
 export_symbol(crypto ${CMAKE_CURRENT_BINARY_DIR}/crypto_p.sym)
 target_link_libraries(crypto ${PLATFORM_LIBS})
 if (WIN32)
-	set(CRYPTO_POSTFIX -${CRYPTO_MAJOR_VERSION})
+	set(CRYPTO_POSTFIX "")
 endif()
 set_target_properties(crypto PROPERTIES
 	OUTPUT_NAME crypto${CRYPTO_POSTFIX}
diff --git a/ssl/CMakeLists.txt b/ssl/CMakeLists.txt
index 3310931..0066bff 100644
--- a/ssl/CMakeLists.txt
+++ b/ssl/CMakeLists.txt
@@ -72,7 +72,7 @@ target_include_directories(ssl
 export_symbol(ssl ${CMAKE_CURRENT_SOURCE_DIR}/ssl.sym)
 target_link_libraries(ssl crypto ${PLATFORM_LIBS})
 if (WIN32)
-	set(SSL_POSTFIX -${SSL_MAJOR_VERSION})
+	set(SSL_POSTFIX "")
 endif()
 set_target_properties(ssl PROPERTIES
 	OUTPUT_NAME ssl${SSL_POSTFIX}
diff --git a/tls/CMakeLists.txt b/tls/CMakeLists.txt
index 9aa10e1..c0547b2 100644
--- a/tls/CMakeLists.txt
+++ b/tls/CMakeLists.txt
@@ -58,7 +58,7 @@ target_include_directories(tls
 export_symbol(tls ${CMAKE_CURRENT_BINARY_DIR}/tls.sym)
 target_link_libraries(tls ${PLATFORM_LIBS})
 if (WIN32)
-	set(TLS_POSTFIX -${TLS_MAJOR_VERSION})
+	set(TLS_POSTFIX "")
 endif()
 set_target_properties(tls PROPERTIES
 	OUTPUT_NAME tls${TLS_POSTFIX}
